test/Kestrel.Performance/AsciiBytesToStringBenchmark.cs

```diff
@@ -0,0 +1,643 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    [Config(typeof(CoreConfig))]
+    public class AsciiBytesToStringBenchmark
+    {
+        private const int Iterations = 100;
+
+        private byte[] _asciiBytes;
+        private string _asciiString = new string('\0', 1024);
+
+        [Params(
+            BenchmarkTypes.KeepAlive,
+            BenchmarkTypes.Accept,
+            BenchmarkTypes.UserAgent,
+            BenchmarkTypes.Cookie
+        )]
+        public BenchmarkTypes Type { get; set; }
+
+        [Setup]
```

Comment:
I think this has been deprecated in favor of `[IterationSetup]`. @natemcmaster just updated the old perf tests.